CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ Previously, setState would only be prioritized at that nesting level so it would
 [W1, S1, W2, S2, S3]
 Now, it will run:
 [W1, S1, S2, S3, W2]
+- If your MvRxView/Fragment does not use any ViewModels, invalidate() will NOT be called in onStart(). In MvRx 1.x, invalidate would be called even if MvRx was not used at all. If you would like to maintain the original behavior, call `postInvalidate()` from onStart in your base Fragment class.
```

Comment:
I _think_ that if the view is dynamically created, like with a RecyclerView, then it needs to be populated (eg via invalidate) before `onViewStateRestored` otherwise scroll position and saved state for items in the list won't be correctly restored.

`invalidate` must be used instead of `postInvalidate` to be synchronous. At airbnb we do it in onActivityCreated in our base fragment. So even if viewmodels are present and MvRx calls postinvalidate for us I think people may still want to manually invoke invalidate earlier

Reply:
@elihart We can recommend something different, this represents the existing behavior though